lib/core/src/obf.cpp

```diff
@@ -289,30 +292,34 @@ obfSavePw( int promptOpt, int fileOpt, int printOpt, const char *pwArg ) {
 
     envVal = obfiGetEnvKey();
 
-    if ( strlen( pwArg ) == 0 ) {
-
+    if ( pwArg == NULL || strlen( pwArg ) == 0 ) {
```

Comment:
more Yoda?

Reply:
Discuss this we did. Change this I will not.